spec/shared_examples/user_reset_password_shared_examples.rb

```diff
@@ -314,17 +316,27 @@
       end
     end
 
-    it 'when change_password! is called, deletes reset_password_token' do
+    it 'when change_password is called, deletes reset_password_token' do
       user.deliver_reset_password_instructions!
 
       expect(user.reset_password_token).not_to be_nil
 
-      user.change_password!('blabulsdf')
-      user.save!
+      user.change_password('blabulsdf')
```

Comment:
Same as above, bang method should be fine here. Interesting that `user.save!` was used but isn't necessary. I should probably look into if that's intentional.